### DISCUSSION TITLE
Question is Enterprise only needed when using commandline?

### DISCUSSION THREAD

**Discussion by Madj4ck** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#341] [General]
Hi,

I might be blind but is Enterprise only needed if using the commandline tool? 
Have tried to search for Enterprise on the wellarchiteched homepage but could not find it.
I don't want to start doing the manual work if at the end that requires Enterprise to.

Sorry if I opened an issue incorrect.

**Comment by VoicuAWS** [comment]
Hi there,

From what I see the Enterprise Edition is required for any deployment methods:

https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/cost-usage-report-dashboards/dashboards/deploy_dashboards/

"After pressing the Sign up button you will be presented with 2 options, please ensure you select the Enterprise Edition during this step"

<img width="717" alt="image" src="https://user-images.githubusercontent.com/99695945/199901278-14529b8b-f92c-4e82-bd1c-bc251993ff94.png">

Hope this helps.
